Controllers/auth/userList.js

```diff
@@ -0,0 +1,27 @@
+import { User } from "../../models";
+
+const userlist = {
+    async list (req, res, next){
+
+        const { page = 2, limit = 10} = req.query;
```

Comment:
these params should be dynamically passed looks like you are taking fixed values destructuring auery prams  should not work like this